### ISSUE TITLE
Sublime text 3 compatibility

### ISSUE THREAD

**Issue by brice** [SublimeText/PhpDoc#11]
Is there any plan to make this plugins compatible with Sublime Text 3?

**Comment by kjda** [comment]
Not working here

ST Build 3065
Ubuntu 14

I installed it using Package manager. I also tried cloning it.
PhpDoc is showing in the "Browse Packages" folder

clicking Ctrl+space just do nothing

**Comment by pablo-pacheco** [comment]
Same problem here

**Comment by Gliptal** [comment]
For me it works with CTRL+Space, but it doesn't with TAB. I'm on Windows 8.

**Comment by toverux** [comment]
@ajberasategui I had exactly the same problem but the shortcut for Alt+/ works. More precisely, the slash key on the numeric pad doesn't works, instead I use Alt+Shift+/ (the slash near the AltGr key).

**Comment by JulianaApolo** [comment]
Same problem here.

**Comment by piersb** [comment]
Identical results to ajberasategui. OSX Yosemite 10.10.5.

I have Quicksilver installed, which is over-riding ^-Space. 
Setting "auto_complete_commit_on_tab": true in user preferences makes no difference.
No joy with Alt-Shift-/ or Alt-/

ST3, package installed via package control

**Comment by jinkrish** [comment]
build:3103 - already there is a snippet in the format `/**,tab` that produces
/**
*
*
*/
installing this is not producing expected doc block. :cry: sad.

**Comment by knifesk** [comment]
Ubuntu 16.04, same problem.. solved by typing ``/**``, then ctrl+space and then go down until the ``/**`` autocomplete and press TAB. Quite impractical though...

**Comment by DrupaListo-com** [comment]
NOTE: I've noticed that after instaling the package via Package Control, you NEED to re-open the current file ( = close it and open it again) and it works after that.

**Comment by smileBeda** [comment]
This behaviour still happens (same as mentioned many times by many other users):
1. OSX 13.3 (22E252)
2. Sublime Text 3 (latest version)

Just type `/**` on top of any function, autocomplete it and all that happens is you get a pop up with hundreds of options to choose from, the first one being `Start Docblock` which inserts a silly 
```
/**
 *
```
(it doesn't even close)

You've another hundred or so options to add `@see` etc, which if chosen, do nothing but add another `*` line to the docblock.

In other words, this is as broken as it gets or, if intended to work like that, as useless as it gets.
Errors show in the console:
```
reloading python 3.3 plugin PhpDoc.CodeDoc
top level value must be an array
reloading settings Packages/User/Preferences.sublime-settings
```

Also ensured that no other package generates the conflict..

Unfortunate!
